src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -251,8 +251,8 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                         var source = remoteProvider.Source;
                         if (source.IsHttp && !source.IsHttps && !source.AllowInsecureConnections)
                         {
-                            await _logger.LogAsync(RestoreLogMessage.CreateWarning(NuGetLogCode.NU1803,
-                                string.Format(CultureInfo.CurrentCulture, Strings.Warning_HttpServerUsage, "restore", source.Source)));
+                            await _logger.LogAsync(RestoreLogMessage.CreateError(NuGetLogCode.NU1804,
```

Comment:
We need a different code, NU18** are warnings. 
https://github.com/NuGet/NuGet.Client/blob/cecbc9a1f7a5cd0ea0a62dea2523f740bbd078d3/src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs#L35-L42. 

We'll need a NU13 code. 
In retrospect, maybe we should've started with a NU13 code. 

Maybe we need to reconsider our structure.